devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -1252,110 +1482,77 @@ void YarpRobotLoggerDevice::run()
     const double time = std::chrono::duration<double>(BipedalLocomotion::clock().now()).count();
 
     std::lock_guard lock(m_bufferManagerMutex);
-    // collect the data
+
     if (m_streamJointStates)
     {
         if (m_robotSensorBridge->getJointPositions(m_jointSensorBuffer))
-        {
```

Comment:
Did you apply the clang format?

Reply:
Fixed in [21bdbdb](https://github.com/ami-iit/bipedal-locomotion-framework/commit/21bdbdbb729890dbd5f758b741f83153aa91cbf3)